c2corg_ui/static/js/documentservice.js

```diff
@@ -181,7 +181,7 @@ app.Document.prototype.isCollaborative = function(type) {
     return false;
   } else if (type === 'outings' || type === 'articles') {
     // check if user == owner or participant of the outing/article
-    return this.auth_.hasEditRights(this.document.associations['users']);
+    return this.auth_.hasEditRights(this.document.associations['users'], null);
```

Comment:
Do you really need to pass a ``null`` value?
``hasEditRights`` should manage itself the case where no ``imageType`` is passed.